File: snippets/fsharp/System/String/IsNullOrWhiteSpace/isnullorwhitespace1.fs
module isnullorwhitespace1.fs
// <Snippet1>
open System

let values = 
    [| null; String.Empty; "ABCDE"
       String(' ', 20); "  \t   "
       String('\u2000', 10) |]

for value in values do
    printfn $"{String.IsNullOrWhiteSpace value}"
// The example displays the following output:
//       True
//       True
//       False
//       True
//       True
//       True
// </Snippet1>

File: snippets/csharp/System/String/IsNullOrWhiteSpace/isnullorwhitespace1.cs
// <Snippet1>
using System;

public class IsNullOrWhiteSpaceExample
{
    public static void Run()
    {
        string[] values = [ null, string.Empty, "ABCDE",
                          new string(' ', 20), "  \t   ",
                          new string('\u2000', 10) ];
        foreach (string value in values)
            Console.WriteLine(string.IsNullOrWhiteSpace(value));
    }
}
// The example displays the following output:
//       True
//       True
//       False
//       True
//       True
//       True
// </Snippet1>
